torch_geometric/explain/algorithm/captum_explainer.py

```diff
@@ -42,8 +60,57 @@ def forward(
         index: Optional[Tensor] = None,
         **kwargs,
     ) -> Explanation:
+        import captum.attr  # noqa
+
+        mask_type = self._get_mask_type()
+
+        inputs, add_forward_args = to_captum_input(
+            x,
+            edge_index,
+            mask_type,
+            *kwargs.values(),
+        )
+
+        if hasattr(model, 'metadata'):
+            metadata = model.metadata
```

Comment:
I think it is safer to use `x.keys()` and `edge_index.keys()`

Reply:
+1.